share/wrf_timeseries.F

```diff
@@ -148,12 +151,16 @@ SUBROUTINE calc_ts_locations( grid )
          CALL wrf_message(message)
    
          ntsloc_temp = 0
+
+         ! Boolean if/else for ij and lat/lon that sets ts_rx(y) to either ij or lat/lon value from tslist
          DO k=1,grid%ntsloc
-         
-            IF (config_flags%map_proj == 0) THEN  ! For idealized cases, no map transformation needed
+            ! Can be ideal case or specified i,j in tslist
+            IF (config_flags%map_proj == 0 .OR. grid%tslist_ij) THEN 
+                ts_rx = grid%itsloc(k)  
+                ts_ry = grid%jtsloc(k)
+            ELSE
                ts_rx = grid%lattsloc(k)           ! NB: (x,y) = (lat,lon) rather than (x,y) = (lon,lat)
                ts_ry = grid%lontsloc(k)
-            ELSE
```

Comment:
The code for setting `ts_rx` and `ts_ry` now looks like:
```fortran
            IF (config_flags%map_proj == 0 .OR. grid%tslist_ij) THEN 
                ts_rx = grid%itsloc(k)  
                ts_ry = grid%jtsloc(k)
            ELSE
               ts_rx = grid%lattsloc(k)           ! NB: (x,y) = (lat,lon) rather than (x,y) = (lon,lat)
               ts_ry = grid%lontsloc(k)
               CALL latlon_to_ij(ts_proj, grid%lattsloc(k), grid%lontsloc(k), ts_rx, ts_ry)
            END IF
```
The `NB` comment no longer makes sense in its new position, and it seems that there's no point to assigning `ts_rx` and `ts_ry` the values from `lattsloc` and `lontsloc`, only to overwrite them with the call to `latlon_to_ij`.

Reply:
@phawbeck 
This is the comment I was referring to that had not been resolved (I don't think - I could be wrong).

Reply:
Ok, thanks - I wasn't sure what the comment was proposing. Remove the `!NB` comment and remove the `ts_rx = ...` and `ts_ry = ...` lines after the `ELSE` statement? Will ts_rx and ts_ry what will be set within the `latlon_to_ij`command?